pds4indextools/pds4_create_xml_index.py

```diff
@@ -715,7 +717,11 @@ def pad_column_values_and_headers(df):
 
     if args.sort_by:
         sort_values = str(args.sort_by).split(',')
-        df.sort_values(by=sort_values, inplace=True)
+        try:
+            df.sort_values(by=sort_values, inplace=True)
+        except KeyError as bad_sort:
+            print(f'Unknown sort key {bad_sort}. For a list of available sort keys, use '
+                  f'the --output-headers-file option.')
```

Comment:
Need to exit the program after this error

Reply:
Does it? In my mind it might be a bit punishing to exit the program and not create an index file because they specified a bad sort key. In this case they can still get their index file, it's just unsorted.

Reply:
Expecting a sorted file and not getting one should be a fatal error. People might miss the error message, or they might be running this in an automated script and thus never know the sort failed, producing the wrong index file results. A bad index file could make it all the way to public release if this error wasn't caught. If the user didn't care about the sort order that much, then they wouldn't have asked for it to be sorted...